### ISSUE TITLE
Unify `PATH` for Windows and Linux

### ISSUE THREAD

**Issue by giggio** [nushell/nushell#9800]
### Related problem

Right now we have to use `PATH` for Linux and `Path` for Windows. It is cumbersome to have a single set a configuration files for both OSs.

### Describe the solution you'd like

Nushell could abstract the `PATH` somehow. Maybe automatically set it correctly, either way you use it.

### Describe alternatives you've considered

I don't know any.

**Comment by fdncred** [comment]
I agree and have said that for a while myself. I think we're just waiting for a someone who is willing to implement it.

**Comment by giggio** [comment]
> `std path add` will do the trick for you. You can import it with `use std "path add"`.

Thanks, @mksinicus, that works!

I read that the standard library is not yet stable, should we assume that this is the way to go, and I close this issue, or maybe this could be implemented somehow? It would be nice to hear from somebody from the Nushell team.

**Comment by fdncred** [comment]
We plan to continue to grow the std lib. In my opinion, I'd like for nushell's internals to treat PATH the same as Path, path, or PaTh. It's just too much of a hassle cross-platform.

**Comment by giggio** [comment]
> We plan to continue to grow the std lib. In my opinion, I'd like for nushell's internals to treat PATH the same as Path, path, or PaTh. It's just too much of a hassle cross-platform.

Ok, @fdncred, I'll leave the issue open, then. Great!

**Comment by cptpiepmatz** [comment]
While we did not unify `PATH` and `Path`, we have all the relevant commands handle both options gracefully. Especially `$env`, as it is case insensitive by default.
